helm/opendistro-es/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: opendistro-es
 sources:
 - https://pages.git.viasat.com/ATG/charts
 # Chart version
-version: 1.13.1
+version: 2.0.0
```

Comment:
The version here represents the current ODFE version. Please use 1.13.1

Reply:
Ok